src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 60px;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 20px;
```

Comment:
this is why the tests don't pass. your first nav__item has a margin on the left and it makes the nav__list wider than it has to be

Reply:
oh i get it so there will be no margin for first child.